test/e2e_node/security_context_test.go

```diff
@@ -274,4 +276,84 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 	})
 
+	Context("when creating containers with AllowPrivilegeEscalation", func() {
+		// parse the docker version
+		out, err := exec.Command("docker", "-v").CombinedOutput()
```

Comment:
Better yet, copy [SkipIfContainerRuntimeIs](https://github.com/kubernetes/kubernetes/blob/afc4506c33063fe64c840f2696af42bfb996436c/test/e2e/framework/util.go#L348) to make `SkipUnlessContainerRuntimeIs`.

This should work for rocket too, right?